Program.cs

```diff
@@ -23,8 +26,15 @@
 StartUpHelper.Run();
 
 var builder = WebApplication.CreateBuilder(args);
+builder.Configuration.AddEnvironmentVariables();
 builder.Services.AddControllersWithViews();
 builder.Services.AddMvc();
+ReverseProxySettings reverseProxySettings = builder.Configuration.GetSection(nameof(ReverseProxySettings)).Get<ReverseProxySettings>();
+
```

Comment:
add comments to the code to say what this is doing

Reply:
done